e2e/test/scenarios/dashboard-cards/dashboard-card-resizing.cy.spec.js

```diff
@@ -279,32 +279,19 @@ describe("scenarios > dashboard card resizing", { tags: "@flaky" }, () => {
 
         cy.request("GET", `/api/dashboard/${dashId}`).then(({ body }) => {
           const dashcards = body.dashcards;
-          dashcards.forEach(({ card }) => {
-            const dashcard = cy.contains(
-              "[data-testid=dashcard-container]",
-              card.name,
-            );
+          dashcards.forEach(({ card }, index) => {
             resizeDashboardCard({
-              card: dashcard,
+              card: getDashboardCard(index),
               x: getDefaultSize(card.display).width * 100,
               y: getDefaultSize(card.display).height * 100,
             });
           });
 
-          saveDashboard();
-          editDashboard();
-
-          dashcards.forEach(({ card }) => {
-            const dashcard = cy.contains(
-              "[data-testid=dashcard-container]",
-              card.name,
-            );
```

Comment:
https://docs.cypress.io/guides/references/best-practices#Assigning-Return-Values

Reply:
Furthermore, I don't get it why did we have to grab cards by their name when we already have the index available. Hence the switch to the battle tested `getDashboardCard(index)`